src/applications/enrollment-verification/components/EnrollmentVerificationLogIn.jsx

```diff
@@ -39,13 +39,11 @@ export function EnrollmentVerificationLogin({ toggleLoginModal }) {
         </a>{' '}
         account now.
       </p>
-      <button
-        type="button"
-        className="usa-button-primary va-button-primary"
+      <va-button
```

Comment:
There is a minor bug with how `va-button` is displaying with `va-alert` here and I have logged an issue with the design system team to look into it:

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2384